Tetris Clone C++/main.cpp

```diff
@@ -61,27 +61,39 @@ int main(int argc, char const *argv[]) {
                     if (event.key.code == sf::Keyboard::Left) {
                         piece.moveLeft();
                     }
-                    if (event.key.code == sf::Keyboard::Right) {
+                    else if (event.key.code == sf::Keyboard::Right) {
                         piece.moveRight();
                     }
-                    if (event.key.code == sf::Keyboard::D) {
-                        piece.rotateClockwise();
-                    }
-                    if (event.key.code == sf::Keyboard::A) {
-                        piece.rotateCounterClockwise();
-                    }
-                    if (event.key.code == sf::Keyboard::Down) {
+                    else if (event.key.code == sf::Keyboard::Down) {
                         autoDrop = false;
                         piece.moveDown();
                         // frame counter to 0 to make it so that
                         // when the player presses down and their piece moves down
                         // it takes a second for the piece to move again automatically
                         frameCounter = 0;
                     }
-                    if (event.key.code == sf::Keyboard::C) {
+                    else if (event.key.code == sf::Keyboard::Up) {
```

Comment:
add a bool to keep track of whether or not the piece can be dropped.

currently, if you hold up, it will just instantly drop pieces, you should have to release the up button before you can drop another piece.